cmake/onnxruntime_unittests.cmake

```diff
@@ -402,7 +402,7 @@ onnxruntime_add_include_to_target(onnx_test_runner_common onnxruntime_common onn
 add_dependencies(onnx_test_runner_common eigen onnx_test_data_proto ${onnxruntime_EXTERNAL_DEPENDENCIES})
 target_include_directories(onnx_test_runner_common PRIVATE ${eigen_INCLUDE_DIRS} ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_BINARY_DIR}/onnx ${ONNXRUNTIME_ROOT})
 set_target_properties(onnx_test_runner_common PROPERTIES FOLDER "ONNXRuntimeTest")
-
+target_link_libraries(onnx_test_runner_common PRIVATE libprotobuf)
```

Comment:
Why do you need this?

Reply:
Ok, as per conversation will add to test_libs.